plugins/modules/azure_rm_appgateway.py

```diff
@@ -2296,6 +2298,11 @@ def exec_module(self, **kwargs):
             else:
                 self.to_do = Actions.NoAction
 
+        update_tags, new_tags = self.update_tags(old_response['tags'])
```

Comment:
lines 2301~2304 wrong indent format. It should be indent by four Spaces

Reply:
Hmm ok. I thought when you said it needed to be it's own conditional you meant completely separate, not a subset of the `self.to_do == Actions.Update` conditional. Fixing.